index.php

```diff
@@ -31,25 +33,31 @@
                             <div class="filter-field-wrapper">
                                 <select>
                                     <option value="1">Date</option>
-                                    <option value="2">Date</option>
-                                    <option value="3">Date</option>
-                                    <option value="4">Type</option>
+                                    <option value="2">Type</option>
                                 </select>
                             </div>
                         </div>
                     </div>
                     <ul class="jobs-listing">
 
-
-                        <li class="job-card">
+<?php
+$sql_jobs = "SELECT id, title, salary, location, date_posted, description FROM jobs";
+$jobs_result = mysqli_query($db_connect, $sql_jobs);
+if (mysqli_num_rows($jobs_result) > 0) {
+  // output data of each row
+    while ($row = mysqli_fetch_assoc($jobs_result)) {
+        $posted_date = date($row["date_posted"]);
+        $posted_date = date_create($posted_date);
```

Comment:
This can be moved inside the `time_message()` function

Reply:
my SQL query is in Index.php so if I move it inside the time_message in functions.php it will be an undefined variable $row["date_posted"]